### PR TITLE
update: Add subscription termination info to Email Aliasing

### PR DESCRIPTION
**Pull request by redoomed1**
Changes proposed in this PR:

- Email Aliasing recommendation page
  - Add subscription termination information to the Email Aliasing page
    - Relevant discussion: https://discuss.privacyguides.net/t/add-subscription-termination-info-to-email-aliasing/19208
    - I also think that there's little reason _not_ to include this information since a similar section is included for each provider on the Email Services page. Also, for a provider like addy, the end result of subscription termination is the same as that of account termination for e.g. Proton Mail: not receiving emails.
  - Remove part of the addy.io description which states that unlimited standard aliases "are less anonymous"
    - Reason: There is already a more detailed description of the limitation of this type of alias under the card.
  - Convert Automatic PGP Encryption "abbreviation" to a footnote with multiple in-text references
- With the introduction of [Proton Duo](https://proton.me/blog/proton-duo) recently, reword the SimpleLogin Premium benefit tied to multiple paid Proton plans
- Make minor grammar changes

<summary>

- [x] I agree to the terms listed below:
  <details><summary>Contribution terms (click to expand)</summary>
  1) I am the sole author of this work.
  2) I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
  3) I have disclosed any relevant conflicts of interest in my post.
  4) I agree to the Community Code of Conduct.
  </details>

### CODE REVIEW

File: includes/abbreviations.en.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@
 *[AOSP]: Android Open Source Project
 *[ATA]: Advanced Technology Attachment
 *[attack surface]: The total number of possible entry points for unauthorized access to a system
+*[Automatic PGP Encryption]: Allows you to encrypt non-encrypted incoming emails before they are forwarded to your mailbox, making sure your primary mailbox provider never sees unencrypted email content.
```
Comment (jonaharagon) [inline includes/abbreviations.en.txt:6]: I don't like this usage of abbreviations, how often is this term used on the page? We should probably use an annotation for the first instance instead: https://squidfunk.github.io/mkdocs-material/reference/annotations/

I know this matches the _current_ behavior, but we might as well improve the situation while we're at it. _Also,_ even if we did want to use an abbreviation, putting one in the abbreviations file is only appropriate when it is in use across multiple pages. If it's only on one page, like in this case, keeping it in the page footer like how it was before is probably better.

Comment (redoomed1) [reply includes/abbreviations.en.txt:6]: > how often is this term used on the page?

3 total times

> putting one in the abbreviations file is only appropriate when it is in use across multiple pages. If it's only on one page, like in this case, keeping it in the page footer like how it was before is probably better.

Ahh, didn't know that. Thanks for the clarification!

File: docs/email-aliasing.md [hunk 2]
```diff
@@ -101,7 +103,7 @@ Notable free features:
 
 SimpleLogin was [acquired by Proton AG](https://proton.me/news/proton-and-simplelogin-join-forces) as of April 8, 2022. If you use Proton Mail for your primary mailbox, SimpleLogin is a great choice. As both products are now owned by the same company you now only have to trust a single entity. We also expect that SimpleLogin will be more tightly integrated with Proton's offerings in the future. SimpleLogin continues to support forwarding to any email provider of your choosing. Securitum [audited](https://simplelogin.io/blog/security-audit) SimpleLogin in early 2022 and all issues [were addressed](https://simplelogin.io/audit2022/web.pdf).
 
-You can link your SimpleLogin account in the settings with your Proton account. If you have the Proton Unlimited, Business, or Visionary Plan, you will have SimpleLogin Premium for free.
+You can link your SimpleLogin account in the settings with your Proton account. If you have the Proton Unlimited, Business, Family, or Visionary plan, you will have SimpleLogin Premium for free.
```
Comment (jonaharagon) [inline docs/email-aliasing.md:106]: The new Proton Duo plan does as well. We might want to just think of a way to word this without enumerating each plan.

Comment (redoomed1) [reply docs/email-aliasing.md:106]:
```suggestion
You can link your SimpleLogin account in the settings with your Proton account. If you have the Proton Unlimited plan or any multi-user Proton plan, you will have SimpleLogin Premium for free.
```

https://proton.me/support/proton-plans#proton-mail-plus

> If you’re looking for a multi-user plan for your family, see the [Proton Family](https://proton.me/support/proton-plans#proton-family) plan below.

> If you want a multi-user plan your can customize for work, see [Proton for Business plans](https://proton.me/support/proton-for-business).

Thank you Proton documentation for providing the succinct wording for this :pray:

File: includes/abbreviations.en.txt (hunk 1 above)
Comment (redoomed1) [reply includes/abbreviations.en.txt:6]: Thoughts on using footnotes like the [forward secrecy definition](https://www.privacyguides.org/en/real-time-communication/#fn:1) on the RTC recommendation page?

Comment (jonaharagon) [reply includes/abbreviations.en.txt:6]: Great idea, I like that 👍

**Review by jonaharagon — APPROVED** [review]

**Review by dngray — APPROVED** [review]

**Pull request merged**